package.json

```diff
@@ -3,15 +3,11 @@
   "version": "0.1.0",
   "description": "MDN Web Docs Style Guide",
   "scripts": {
-    "build": "npx @11ty/eleventy --pathprefix=mdn-fiori --input=styleguide --output=docs",
-    "build:sass": "node-sass-chokidar ./node_modules/mdn-minimalist/sass/ -o ./styleguide/css -w",
-    "build:sass:styleguide": "node-sass-chokidar ./styleguide/sass/ -o ./styleguide/css/styleguide -w",
-    "build:sass:prod": "node-sass-chokidar ./node_modules/mdn-minimalist/sass/ -o ./styleguide/css",
-    "build:sass:styleguide:prod": "node-sass-chokidar ./styleguide/sass/ -o ./styleguide/css/styleguide",
-    "build:production": "run-s copy:typography build:sass:prod build:sass:styleguide:prod build",
-    "copy:typography": "cp -R ./node_modules/mdn-minimalist/typography/ ./styleguide/typography/",
-    "serve": "npx @11ty/eleventy  --input=styleguide --output=docs --serve",
-    "start": "run-p copy:typography build:sass build:sass:styleguide serve"
+    "copy:sass": "cp -R ./node_modules/mdn-minimalist/sass/ ./sass",
+    "copy:typography": "cp -R ./node_modules/mdn-minimalist/typography/ ./typography",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
+    "start": "npm-run-all copy:sass copy:typography storybook"
```

Comment:
Do we still need  to run `copy:sass` if we're importing `mdn-minimalist` from node_modules? I think we do need `copy:typography` for now (until we add install something like `resolve-url-loader`) otherwise we run into the reference errors you mentioned earlier.

Reply:
We probably do not. Let me remove that one.